apps/ui/src/views/My/Notifications.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import { _rt } from '@/helpers/utils';
+
+const notificationsStore = useNotificationsStore();
+const { setTitle } = useTitle();
+
+watchEffect(async () => {
+  setTitle(
+    `Notifications${notificationsStore.unreadNotificationsCount ? ` (${notificationsStore.unreadNotificationsCount} unread)` : ''}`
+  );
+});
+
+onUnmounted(() => notificationsStore.markAllAsRead());
```

Comment:
Should be good with a `beforeunload` event.

This method still has some drawback, as it's not being triggered on mobile in some cases, when app is being closed instead of browser tab